### PR TITLE
Improve karate's tests reliability and project's README

### PR DESCRIPTION
**Pull request by GabrielFleischer**
Some tests defined in karate are failing because they are buggy.

This PR tries to fix those issues

It also adds badges and links to the test results in the project's README. Currently, the badge is blank as the action does not exist yet.

### CODE REVIEW

File: .github/workflows/karate/be1-go.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
-name: Integration Tests
+name: Integration - Go
```
Comment (pierluca) [inline .github/workflows/karate/be1-go.yaml:1]: Are you sure you can move this file in a subfolder?

File: tests/karate/src/test/java/be/constants.feature [hunk 2]
```diff
@@ -236,9 +229,9 @@ Feature: Constants
     * def getValidRegisteredVotes = call createValidRegisteredVotes
     * def getInvalidRegisteredVotes = call createInvalidRegisteredVotes
 
-    * def getCoinIssuancePubKey = call createIssuancePubKey
-    * def getCreateSignatureForCoinIssuance = call createSignatureForCoinIssuance
-    * def getValidOutputs =   [{ "value": 32, "script": { "type": "P2PKH", "pubkey_hash": "2jmj7l5rSw0yVb-vlWAYkK-YBwk=" } }]
+    * def getCoinIssuancePubKey = call organizerPk
```
Comment (pierluca) [inline tests/karate/src/test/java/be/constants.feature:232]: Are you sure about this change? It seems contrary to the spec referenced in the original test implementation 🤔

**Review by pierluca — COMMENTED** [review]
Thanks for looking into this.
I do have some concerns about the digital cash changes. Can we maybe have a quick chat about those?

File: tests/karate/src/test/java/be/constants.feature (hunk 2 above)
Comment (GabrielFleischer) [reply tests/karate/src/test/java/be/constants.feature:232]: As discussed, yes

File: .github/workflows/karate/be1-go.yaml (hunk 1 above)
Comment (GabrielFleischer) [reply .github/workflows/karate/be1-go.yaml:1]: Well, you're right.

It does not seem to work

**Review by pierluca — APPROVED** [review]

**Pull request merged**